integration_tests/ctesque/src/test/java/android/content/res/ResourcesTest.java

```diff
@@ -439,6 +439,12 @@ public void testGetBitmapDrawableForUnknownId() {
     assertThat(resources.getDrawable(Integer.MAX_VALUE)).isInstanceOf(BitmapDrawable.class);
   }
 
+  @Test
```

Comment:
Can you change this test to take density into account? We run this test on devices as well:

```java
  @Test
  public void testGetNinePatchDrawableIntrinsicWidth() {
    Float density = resources.getDisplayMetrics().density;
    NinePatchDrawable ninePatchDrawable =
        (NinePatchDrawable) resources.getDrawable(R.drawable.nine_patch_drawable);
    assertThat((float) ninePatchDrawable.getIntrinsicWidth()).isEqualTo(98.0f * density);
  }
```